packages/docs/versioned_docs/version-2.0.0-beta-3/welcome.md

```diff
@@ -31,8 +30,13 @@ Attesters are at the application layer. They are the platforms, businesses, and
 UniRep's key offering is providing developers with a pre-built, audited system to create apps that securely handle private user data and manage reputation using ZK technology. UniRep protocol allows applications to securely attest to user data changes through the use of dynamic, short-lived identifiers known as **epoch keys**.
 
 :::
+### Application built by our core contributors ###
 
-### Few inspriations for you to get started
+- Built by [Kittybest](https://github.com/kittybest),  Web2 Authentication Bridging: [My-Badge](https://github.com/kittybest/my-badge)
+- Built by [CJ](https://github.com/CJ-Rose) ,  Craiglist reinvent: [Trustlist](https://github.com/CJ-Rose/trustlist)
```

Comment:
Unirep social can be included?
and like https://github.com/Unirep/zketh and https://github.com/vimwitch/zkdao
I think they are all good examples